static/js/consumers.js

```diff
@@ -0,0 +1,57 @@
+import * as Table from './table.js'
+import * as HTTP from './http.js'
+import * as DOM from './dom.js'
+
+const vhost = window.sessionStorage.getItem('vhost')
+let url = 'api/consumers'
+if (vhost && vhost !== '_all') {
+  url = `api/consumers/${encodeURIComponent(vhost)}`
+}
+
+const tableOptions = {
+  url,
+  keyColumns: ['channel_details', 'consumer_tag'],
+  interval: 5000,
+  pagination: true,
+  columnSelector: true,
+  search: true
+}
+
+Table.renderTable('table', tableOptions, function (tr, item, all) {
```

Comment:
the third parameter to the callback is weather the row is rendered the first time or is being updated. As no columns are being changed (rows are only added or removed), we don't need to re-render the cells.

```suggestion
Table.renderTable('table', tableOptions, function (tr, item, firstRender) {
  if (!firstRender) return
```